.github/workflows/testsuite.yaml

```diff
@@ -163,11 +158,11 @@ jobs:
 
       - name: Install
         run: |
-          pip install --no-deps .
+          pip install --no-deps -e .
```

Comment:
Should this be a developer install?

Reply:
Yes, I want one of the tests to run with a developer install to cover a part of the `__init__.py` that is not covered for standard installs.